_src/static/sass/components/_headings.scss

```diff
@@ -0,0 +1,99 @@
+// ------------------------------
+// edX Pattern Library: Components - Headings
+
+// About: Contains base styling for headings, relying on Bourbon's mathematical scales for precision.
+
+// Scale:
+// 12, 14, 16, 18, 21, 24, 32, 43
+// ----------------------------
+
+// #GLOBAL
+// #INDIVIDUAL HEADINGS
+// ----------------------------
+
+
+// ----------------------------
+// #GLOBAL
+// ----------------------------
+.hd-1,
+.hd-2,
+.hd-3,
+.hd-4,
+.hd-5,
+.hd-6 {
+    @extend %reset-headings;
```

Comment:
Should we reset margins at this level or at the extends level rather than the ``margin: x x Y x`` shorthand syntax? If so, for particular headings we can just use the ``margin-bottom: Y`` property.